vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Exhibition.java

```diff
@@ -0,0 +1,49 @@
+package edu.asu.diging.vspace.core.model.impl;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IExhibition;
+
+@Entity
+public class Exhibition extends VSpaceElement implements IExhibition {
+
+	@Id
+	@GeneratedValue(generator = "id-generator")
+	@GenericGenerator(name = "id-generator", 	
```

Comment:
you can choose the generator id more or less as you please as long as it is unique (that there are two generators with the same name already doesn't mean it's right ;)